modules/reference/pages/data-transform-golang-sdk.adoc

```diff
@@ -20,6 +21,14 @@ The `OnRecordWritten` function registers a callback of type <<onrecordwrittencal
 
 The function should be called in a package's `main` function to register the transform function that will be applied.
 
+==== ToTopic
+
+```go
+func ToTopic(topic string) WriteOpt
```

Comment:
Sidenote: I think this is a change to the API (@rockwotj ?) which should be documented in 'incompatible changes' in 24.1 (even though WASM was in beta in 23.3 we should still note this breaking change).

Reply:
It's not incompatible, it's purely additive, but you will have to upgrade the SDK

Reply:
@rockwotj do we need to provide guidance on upgrading? In the [prereqs](https://deploy-preview-374--redpanda-docs-preview.netlify.app/current/develop/data-transforms/run-transforms/#prerequisites) we say "A Redpanda cluster running version 23.3," could we change that to "23.3 and later"?

Reply:
I think there is an open question of if we should be versioning the SDK docs or not, but these APIs will only be able to be used in 24.1